feature-flags/README.md

```diff
@@ -17,7 +17,16 @@ of 3 categories:
 - For configuration toggles, prefer environment variable backed feature flags
 - Avoid creating feature flags with LaunchDarkly. This is considered deprecated
   unless used in React during our transition away from React.
-- Prefer using a feature flag when implementing a change which impacts SOPs.
+- Prefer using feature flags by default unless there's a compelling reason not to.
+  - Examples for feature flag use:
+    - SOPs are impacted
+    - Training required
+    - Release flexibility
+  - Examples where feature flags may not be needed:
+    - Label changes
```

Comment:
Not sure I would include this as many of the label changes can impact SOPs, and it is hard to know what is in our customer SOPs.